src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/testext/WebServerRecipe.java

```diff
@@ -0,0 +1,127 @@
+package jp.ikedam.jenkins.plugins.updatesitesmanager.testext;
+
+import jp.ikedam.jenkins.plugins.updatesitesmanager.ManagedUpdateSiteJenkinsTest;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.StringUtils;
+import org.jvnet.hudson.test.JenkinsRecipe;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.mortbay.jetty.HttpConnection;
+import org.mortbay.jetty.Server;
+import org.mortbay.jetty.bio.SocketConnector;
+import org.mortbay.jetty.handler.AbstractHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
(Note for me)
slf4j is included via jenkins-war.
jenkins-core starts to use slf4j since 1.628.